src/Microsoft.AspNetCore.Mvc.Razor.Extensions/ViewComponentTagHelperPass.cs

```diff
@@ -37,9 +35,9 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument, DocumentIRNo
                 }
             }
 
-            foreach (var (node, parent) in visitor.CreateTagHelpers)
+            foreach (var (parent, node) in visitor.CreateTagHelpers)
```

Comment:
Why? `foreach (var nodeReference in visitor.CreateTagHelpers)` seems super clean.